### PR TITLE
Add undocumented DGU-related repositories

### PR DESCRIPTION
**Pull request by ChrisBAshton**
See commits for details.

Trello: https://trello.com/c/tsBL9lPH/3161-work-through-list-of-missing-repos

### CODE REVIEW

**Review by kentsanggds — COMMENTED** [review]
This will probably be archived as I have moved the docker code to the ckanext-datagovuk repo

File: data/repos.yml [hunk 1]
```diff
@@ -264,6 +264,12 @@
   team: "#data-products"
   type: Data science
 
+- repo_name: docker-ckan
```
Comment (kentsanggds) [inline data/repos.yml:267]: This will probably be archived as I have moved the docker code to the ckanext-datagovuk repo

File: data/repos.yml [hunk 2]
```diff
@@ -270,6 +270,12 @@
   sentry_url: false
   dashboard_url: false
 
+- repo_name: docker-collectd-plugin
```
Comment (kentsanggds) [inline data/repos.yml:273]: Not sure this is part of DGU, I certainly haven't used it and it hasn't been touch in almost 2 years, maybe another repo to archive

File: data/repos.yml [hunk 3]
```diff
@@ -307,6 +307,12 @@
   team: "#tech-content-interactions-on-platform-govuk"
   production_hosted_on: eks
 
+- repo_name: fake-ckan_harvest_source
```
Comment (kentsanggds) [inline data/repos.yml:310]: This repo should probably be archived too as it's superseded by clan-mock-harvest-sources repo

Comment (ChrisBAshton) [reply data/repos.yml:310]: I see you've now archived it. Thanks!

File: data/repos.yml (hunk 2 above)
Comment (ChrisBAshton) [reply data/repos.yml:267]: I'll take it out of this PR so it can be looked at separately.

**Review by kentsanggds — APPROVED** [review]
There appears to be some kind of conflict with main but otherwise LGTM

**Pull request merged**